torch/fx/graph.py

```diff
@@ -157,8 +159,20 @@ def node_copy(self, node: Node, arg_transform: Callable[[Node], Argument] = lamb
         return self.create_node(node.op, node.target, args, kwargs, name)
 
     def output(self, result: Argument):
-        self.result = result
         self._mark_uses(result)
+        return self.create_node(op='output', target='output', args=(result,))
+
+    @property
```

Comment:
I feel like this function should not exist in the version where `output` is not special. Especially as a property it suggests it is fast, but is actually linear in the size of the graph.

Reply:
I agree with that, but one thing this is hiding is the fact that the output value is actually `args[0]` in the `output` node. I don't think that's nice and people will probably mess that up a lot if they have to manually extract the output value